src/sweetalert2.scss

```diff
@@ -758,56 +764,47 @@ body {
   }
 }
 
-.swal2-progresssteps {
-  $lightblue: #add8e6;
-  $blue: #3085d6;
-
+.swal2-progress-steps {
   align-items: center;
   margin: $swal2-progress-steps-margin;
-  padding: 0;
-  font-weight: 600;
+  padding: $swal2-progress-steps-padding;
+  background: $swal2-progress-steps-background;
+  font-weight: $swal2-progress-steps-font-weight;
 
   li {
     display: inline-block;
     position: relative;
   }
 
-  .swal2-progresscircle {
-    width: 2em;
-    height: 2em;
-    border-radius: 2em;
-    background: $blue;
-    color: $swal2-white;
-    line-height: 2em;
+  .swal2-progress-step {
+    width: $swal2-progress-steps-step-width;
+    height: $swal2-progress-steps-step-height;
+    border-radius: $swal2-progress-steps-step-border-radius;
+    background: $swal2-progress-steps-active-step-background;
+    color: $swal2-progress-steps-active-step-color;
+    line-height: $swal2-progress-steps-step-height;
     text-align: center;
```

Comment:
Sorry to be picky, but i'm still not convinced about the `steps-step` repetition in the class name. I understand that when applied it will be applied to one single step, but since you cannot have one step style different from the others, then the property name should be _plural_ and not _singular_.  So for example  `$swal2-progress-steps-step-height` should be `$swal2-progress-steps-height`, because all the steps share this same style.

Reply:
done, https://github.com/sweetalert2/sweetalert2/pull/1411/commits/d828ced7e7d5b242d08a229ec70da861ff390aae